### PR TITLE
Keba: Update discovery mechanism

### PR DESCRIPTION
**Pull request by t-mon**
nymea-plugins pull request checklist:

- [x] Make sure the pull request's title is of format "Plugin name: Add support for xyz" or "New plugin: Plugin name"

- [x] Did you test the changes on hardware, if not (e.g. absence of required hardware), please mention a person to confirm it has been tested.

- [x] Did you update the plugin's README.md accordingly?

- [x] Did you update translations (`cd builddir && make lupdate`)?

- [x] If you added a new plugin, should it be added to nymea-plugins-all or nymea-plugins-maker?

### CODE REVIEW

**Comment by mzanetti** [comment]
Fixes #512

File: keba/integrationpluginkeba.cpp [hunk 1]
```diff
@@ -358,6 +378,11 @@ void IntegrationPluginKeba::searchNetworkDevices()
     });
 }
 
+void IntegrationPluginKeba::onDiscoveryWaitUpdResponseTimeout()
+{
+
+}
+
```
Comment (mzanetti) [inline keba/integrationpluginkeba.cpp:385]: leftover

File: keba/integrationpluginkeba.cpp [hunk 2]
```diff
@@ -605,8 +630,12 @@ void IntegrationPluginKeba::executeAction(ThingActionInfo *info)
 
         QUuid requestId;
         if (action.actionTypeId() == wallboxMaxChargingCurrentActionTypeId) {
-            int milliAmpere = action.param(wallboxMaxChargingCurrentActionMaxChargingCurrentParamTypeId).value().toUInt() * 1000;
+            int milliAmpere = action.paramValue(wallboxMaxChargingCurrentActionMaxChargingCurrentParamTypeId).toUInt() * 1000;
             requestId = keba->setMaxAmpereGeneral(milliAmpere);
+            // Note: since the response only indicates the successfull receiving of the command,
```
Comment (mzanetti) [inline keba/integrationpluginkeba.cpp:635]: successful, one l

**Review by mzanetti — COMMENTED** [review]
If I read this correctly it still has the issue that it won't work any more once the DHCP lease expires, right?

**Comment by t-mon** [comment]
> If I read this correctly it still has the issue that it won't work any more once the DHCP lease expires, right?

yes, we want to use the network device monitor from the core once implemented here. No point in adding it now and then remove it again...

**Review by mzanetti — APPROVED** [review]

**Pull request closed without merging**